src/View/ProgressWindow.py

```diff
@@ -1,8 +1,13 @@
 import threading
+import platform
```

Comment:
Unused import (and `from pathlib import Path` & `from src.Controller.PathHandler import resource_path`